### PR TITLE
REFPLTV-2563  [VTS]dsHostInit and dsHostTerm not defined properly

### PR DESCRIPTION
**Pull request by aarthikasuresh12-ai**
dsHostInit
dsHostTerm
dsGetCPUTemperature
are updated as per test script
And dsGetSocIDFromSDK is not supported for rpi

### CODE REVIEW

File: dsHost.c [hunk 1]
```diff
@@ -29,13 +29,19 @@ extern "C" {
...
+static bool host_initialized = false;
 #define FILE_SIZE 50
 #define SIZE 10
 dsError_t dsHostInit()
 {
     dsError_t ret = dsERR_NONE;
-
-    return ret;
+    if (host_initialized) { 
+        dsError_t ret = dsERR_ALREADY_INITIALIZED;
```
Comment (arun-madhavan-013) [inline dsHost.c:39]: Please remove unnecessary declarations.

File: dsHost.c [hunk 2]
```diff
@@ -58,6 +64,15 @@ dsError_t dsGetCPUTemperature(float *cpuTemperature)
     char Temp_File[FILE_SIZE];
     char temp_value[SIZE];
     FILE *fp = NULL;
+    if (!host_initialized) {                                                                                                                                               
+    dsError_t ret = dsERR_NOT_INITIALIZED;
```
Comment (arun-madhavan-013) [inline dsHost.c:68]: Please remove unnecessary declaration; you can return the error code directly right ?

**Review by arun-madhavan-013 — CHANGES_REQUESTED** [review]

File: dsHost.c (hunk 2 above)
Comment (aarthikasuresh12-ai) [reply dsHost.c]: removed unnecessary declaration

File: dsHost.c [hunk 3]
```diff
@@ -159,19 +169,30 @@ dsError_t dsGetTotalSystemGraphicsMemory(uint64_t* memory)
...
 }
 
-dsError_t dsGetHostEDID(unsigned char *edid, int *length) {
-        return dsERR_OPERATION_NOT_SUPPORTED;
+dsError_t dsGetHostEDID(unsigned char *edid, int *length) 
+{
+    if (!host_initialized) {                                                                                                                                                                                                                                                                        
+        return dsERR_NOT_INITIALIZED;                                                                                                                                                     
+    }  
+	if (edid == NULL || length == NULL ) {
```
Comment (deepthi-ps) [inline dsHost.c:184]: Please correct the alignment

**Review by arun-madhavan-013 — APPROVED** [review]
Logical issues are corrected in this.

**Pull request merged**